.styleci.yml

```diff
@@ -2,7 +2,6 @@ preset: symfony
 
 enabled:
   - alpha_ordered_imports
-  - short_array_syntax
```

Comment:
Else StyleCI Errors: https://github.styleci.io/analyses/EAVJNA?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
thanks. seems that using the symfony preset is occasionally creating problems like this one.